File: Python/cuentaDriver.py
from cuenta import Cuenta
from uber_car import Uber_car

class CuentaDriver(Cuenta):
    carro = Uber_car("","")

    def __init__(self, id, name, document, email, password, number, carro):
        super().__init__(id, name, document, email, password, number)
        self.carro = carro

File: Java/CuentaDriver.java
class CuentaDriver extends Cuenta{
    Uber_cars carro;

    public CuentaDriver(Integer id, String name, String document, String email, String password, Integer number, Uber_cars carro){
        super(id, name, document, email, password, number);
        this.carro = carro;
    }
}
